app/Services/PackageRoutingFieldCatalog.php

```
@@ -0,0 +1,218 @@
+    /**
+     * @return array<string, string>
+     */
+    public function matchModeOptions(): array
+    {
+        return self::MATCH_MODES;
+    }
+
+    /**
+     * @return array<string, string>
+     */
+    public function customFieldOptions(?int $companyId = null): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$companyId'. (undefined)

(UnusedFormalParameter)